bootstrap.sh

```diff
@@ -2,13 +2,13 @@
 set -e
 set -x
 
-# overwrite env from base image
+# Overwrite env from base image
 export HOME=/root
 export TEMP_DIR=/tmp
 export TMPDIR=/tmp
 export TMP_DIR=/tmp
 
-#install dependent libraries
+# Install dependent libraries
```

Comment:
Do we really need image magick in the build image? It's not being used at all and I don't see it being saved with the bundle.